net/shorewall-lite/Makefile

```diff
@@ -14,7 +14,7 @@ PKG_DIRECTORY:=5.1
 PKG_MAINVERSION:=5.1.4
 PKG_RELEASE:=1
 
-PKG_SOURCE_URL:=http://www.shorewall.net/pub/shorewall/$(PKG_DIRECTORY)/shorewall-$(PKG_MAINVERSION)/ 
+PKG_SOURCE_URL:=http://www.shorewall.net/pub/shorewall/$(PKG_DIRECTORY)/shorewall-$(PKG_MAINVERSION)/
```

Comment:
Changes a different package, so should go in a separate commit. Even if it's for the same software.

Reply:
Done